base/pcg-random/pcg_extras.hpp

```diff
@@ -1,3 +1,5 @@
+// NOLINTBEGIN(*)
```

Comment:
Can it be just `NOLINTBEGIN` for all checks, w/o braces ?

Reply:
According to their docs [0], unfortunately not.

(Btw., I have used NOLINTBEGIN/NOLINTEND here because the file was copypasted from elsewhere (i.e. third-party code) and I didn't want to litter it with "fine-granular" exclusions.)

[0] https://clang.llvm.org/extra/clang-tidy/